### DISCUSSION TITLE
Combine Android TV & Raspotify on Raspberry Pi 4

### DISCUSSION THREAD

**Discussion by cfkmo** [dtcooper/raspotify#537] [Q&A] [answered]
Hi everyone.

The raspberry pi topic is relatively new to me and currently I am trying to get my first "project" done :) As it seems, my topic is somehow special, I couldn't find anything about it and therefore I am asking here if my plan can work?!

So I want to combine Android TV for the usage of Netflix, Amazon Prime ... with Spotify Connect (raspotify) to enable my old amplifier to be "smart"/ wlan speaker.

For aTV I used LineageOS 18.1 ATV from **konstakang.com/devices/rpi4/LineageOS18-ATV/** . So is there any possibility to make Raspotify work on the same Pi?

greetings

**Comment by JasonLG1979** [comment, marked as answer]
No. Raspoify is librespot compiled and packaged as a .deb for Debian based systems. You would need to build a librespot binary for Android.

**Reply by JasonLG1979** [reply]
It would be easier to just install the Spotify Android app. It has Spotify Connect baked into it already.

**Reply by cfkmo** [reply]
Thanks for your first replies. Are there any differences for the usage if I install the Android App instead of Raspotify? Any functionalities missing or something?!

**Reply by JasonLG1979** [reply]
No the opposite. Librespot is a reverse engineering of Spotify Connect. The Connect implementation in the Android app, being written by Spotify, will naturally be more spec compliant and feature complete.

**Reply by JasonLG1979** [reply]
As far as use, the Spotify app would need to be running in the background I would think? I don't know, I don't use the Android app in that way but I know the functionality exists.
